test/integration/kubernetes_deploy_test.rb

```diff
@@ -340,6 +341,16 @@ def test_output_of_failed_unmanaged_pod
     ], in_order: true)
   end
 
+  def test_deployment_includes_ejson_secrets
```

Comment:
What is this proving that `test_create_and_update_secrets_from_ejson` (in the other file) doesn't already cover?

Reply:
Nothing really - I started with this test and `resources += secrets_from_ejson` and worked backwards, this can be removed now.